### PR TITLE
Add rerun in Grinder on the same machine link

### PR DESCRIPTION
**Pull request by llxia**
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/465/
image

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -858,16 +858,23 @@ def uploadToArtifactory(pattern) {
...
 		i++;
+		if ( key == "LABEL" ) {
+			labelValue = "LABEL=${value}"
+		}
 	}
 	echo "Rerun in Grinder: ${url}"
-	currentBuild.description += "<br><a href=${url}>Rerun in Grinder</a> TARGET can be changed to run only the failed test target"
+	currentBuild.description += "<br><a href=${url}>Rerun in Grinder</a> TARGET to run only the failed test targets"
+	def sameMachineUrl = url.replace(labelValue,"LABEL=${NODE_NAME}")
+	echo "Rerun in Grinder on the same machine: ${sameMachineUrl}"
```
Comment (smlambert) [inline buildenv/jenkins/JenkinsfileBase:876]: could drop the 'the' to read to save on a few characters if you want:
"Rerun in Grinder on same machine"

**Review by smlambert — APPROVED** [review]
thanks @llxia - some minor update suggestions, but otherwise LGTM

File: buildenv/jenkins/JenkinsfileBase [hunk 2]
```diff
@@ -858,16 +858,23 @@ def uploadToArtifactory(pattern) {
...
 			url += "&amp;"
 		}
 		i++;
+		if ( key == "LABEL" ) {
+			labelValue = "LABEL=${value}"
+		}
 	}
 	echo "Rerun in Grinder: ${url}"
-	currentBuild.description += "<br><a href=${url}>Rerun in Grinder</a> TARGET can be changed to run only the failed test target"
+	currentBuild.description += "<br><a href=${url}>Rerun in Grinder</a> TARGET to run only the failed test targets"
```
[the remark this answers is not in the document]
Comment (llxia) [reply buildenv/jenkins/JenkinsfileBase]: Added the word Change and Grinder wiki link

**Comment by llxia** [comment]
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/472/

image

**Review by renfeiw — APPROVED** [review]

**Pull request merged**